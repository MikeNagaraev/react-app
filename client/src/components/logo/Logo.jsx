import React from "react";

export default (props) => React.createElement("h1", { className: props.className }, props.title);