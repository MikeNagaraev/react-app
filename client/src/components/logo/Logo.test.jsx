import * as React from 'react';
import { mount, configure } from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';
import renderer from 'react-test-renderer';

import Logo from './Logo';

describe('Logo component', () => {
    beforeAll(() => {
        configure({ adapter: new Adapter() });
    });

    it('should render component', () => {
        const enzymeWrapper = mount(<Logo />);
        expect(enzymeWrapper.find('h1')).toHaveLength(1);
    });

    it('renders correctly snapshot', () => {
        const tree = renderer.create(
            <Logo title="netfilxroulette" className="logo" />
        ).toJSON();
        expect(tree).toMatchSnapshot();
    });
});
