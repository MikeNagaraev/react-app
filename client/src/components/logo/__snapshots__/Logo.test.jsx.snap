// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Logo component renders correctly snapshot 1`] = `
<h1
  className="logo"
>
  netfilxroulette
</h1>
`;
